commands/doit.go

```diff
@@ -44,49 +44,32 @@ var DoitCmd = &Command{
 	},
 }
 
-// Context holds the current auth context
-var Context string
-
-// APIURL holds the API URL to use.
-var APIURL string
-
-// Token holds the global authorization token.
-var Token string
-
-// Output holds the global output format.
-var Output string
-
-// Verbose toggles verbose output.
-var Verbose bool
-
-var requiredColor = color.New(color.Bold).SprintfFunc()
-
-// Writer is where output should be written to.
-var Writer = os.Stdout
-
-// Trace toggles http tracing output.
-var Trace bool
-
-// cfgFile is the location of the config file
-var cfgFile string
-
-// cfgFileWriter is the config file writer
-var cfgFileWriter = defaultConfigFileWriter
-
-// ErrNoAccessToken is an error for when there is no access token.
-var ErrNoAccessToken = errors.New("no access token has been configured")
+var (
```

Comment:
thank you for this!

Reply:
couldn't take it any more. Especially exported intermingled with local.